subsys/bluetooth/host/conn.c

```diff
@@ -226,8 +267,9 @@ static void tx_notify(struct bt_conn *conn)
 
 		key = irq_lock();
 		if (!sys_slist_is_empty(&conn->tx_complete)) {
-			tx = CONTAINER_OF(sys_slist_get_not_empty(&conn->tx_complete),
-					  struct bt_conn_tx, node);
+			const sys_snode_t *node = sys_slist_get_not_empty(&conn->tx_complete);
+
+			tx = CONTAINER_OF(node, struct bt_conn_tx, node);
 		}
```

Comment:
Just for my own education: How is this different than

```c
tx = sys_slist_get(&conn->tx_complete);
```

Reply:
not sure. I just added a temp var to aid readability.